src/docs/processing-tickets.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: "Processing Issues & PRs"
+---
+
+This document describes Sentry's process for handling issues and PRs—let's use "tickets" to refer to both together, if that's okay—that are created by external contributors in our public GitHub repos.
+
+## 1. Validate
+
+*SLA: 1 hour*
+
+The Open Source Team maintains [a bot](https://github.com/getsentry/.github/blob/main/.github/workflows/validate-new-issue.yml) that validates incoming tickets. Invalid tickets are closed; valid tickets are given the label `Status: Unrouted`. The bot is installed on three repos: `sentry`, `sentry-docs`, and `onpremise`.
+
+## 2. Route
+
+*SLA: 72 hours*
+
+Each ticket is routed to a single Engineering team based on our internal [Product and Service Registry](https://www.notion.so/Product-and-Service-Registry-f90c702816794dcaa7f81ef8e79e40ea):
```

Comment:
We should consider making this registry public too.

Reply:
Agreed.